tests/scene/test_packed_scene.h

```diff
@@ -95,16 +100,24 @@ TEST_CASE("[PackedScene] Signals Preserved when Packing Scene") {
 		CHECK_EQ(state->get_connection_count(), 3);
 	}
 
-	/*
-	// FIXME: This subcase requires GH-48064 to be fixed.
+	/* TODO needs fixed
```

Comment:
Closing the issue does not fix this case?

Reply:
I started working on `tests/scene/test_packed_scene.h` but decided to wait for feedback on the rest of the code because I wasn't sure if the solution in this PR would be too complex.

so `tests/scene/test_packed_scene.h` should be ignored for now

Reply:
The tests were for a solution that used `CONNECT_INHERITED`, so they won't work for this PR now